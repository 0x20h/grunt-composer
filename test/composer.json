{
    "name": "voce/grunt-composer",
    "description": "Tests for grunt-composer",
    "authors": [
        {
            "name": "Michael Pretty",
            "email": "mpretty@voceconnect.com"
        }
    ],
    "repositories": [
        {
            "type": "artifact",
            "url": "fixtures"
        }
    ],
    "require": {
        "voce/grunt-composer-test": "0.1.0"
    },
    "config": {
        "vendor-dir": "tmp"
    }
}
