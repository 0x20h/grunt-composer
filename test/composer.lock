{
    "_readme": [
        "This file locks the dependencies of your project to a known state",
        "Read more about it at http://getcomposer.org/doc/01-basic-usage.md#composer-lock-the-lock-file"
    ],
    "hash": "f746595285c1c97fa7f44428720e05a7",
    "packages": [
        {
            "name": "voce/grunt-composer-test",
            "version": "0.1.0",
            "dist": {
                "type": "zip",
                "url": "/Users/prettyboymp/projects/grunt-composer/test/fixtures/voce-grunt-composer-test.0.1.0.zip",
                "reference": "voce-grunt-composer-test.0.1.0.zip",
                "shasum": "630135aa7fb68a97631a066b02c2ebd3342f9b45"
            },
            "type": "library",
            "authors": [
                {
                    "name": "Michael Pretty",
                    "email": "mpretty@voceconnect.com"
                }
            ],
            "description": "Tests repo"
        }
    ],
    "packages-dev": [

    ],
    "aliases": [

    ],
    "minimum-stability": "stable",
    "stability-flags": [

    ],
    "platform": [

    ],
    "platform-dev": [

    ]
}
