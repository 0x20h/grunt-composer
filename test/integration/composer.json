{
    "name": "voce/grunt-composer",
    "description": "Tests for grunt-composer",
    "authors": [
        {
            "name": "Michael Pretty",
            "email": "mpretty@voceconnect.com"
        }
    ],
    "config": {
        "vendor-dir": "tmp"
    }
}