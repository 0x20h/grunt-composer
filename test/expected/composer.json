{
    "name": "voce/grunt-composer-test",
    "description": "Tests repo",
    "version": "0.1.0",
    "authors": [
        {
            "name": "Michael Pretty",
            "email": "mpretty@voceconnect.com"
        }
    ]
}
